src/public/Interfaces.ts

```diff
@@ -50,11 +49,11 @@ export interface IContext {
      * @param err A user-defined error to pass back to the runtime. If present, your function execution will fail.
      * @param result An object containing output binding data.
      */
-    done: IDoneCallback;
+    done(err?: any, result?: any): void;
```

Comment:
duplicate of DoneCallback type?

Reply:
Duplicate, but because Interfaces.ts is the only part that we're going to publish (better control of what we expose)... I didn't want to keep it as a type "DoneCallback" on context because I wanted done to show up as a method and not a property in intellisense. Thoughts?